Sources/SwiftKueryPostgreSQL/PostgreSQLConnection.swift

```diff
@@ -586,3 +572,89 @@ public class PostgreSQLConnection: Connection {
         return nil
     }
 }
+
+class PostgreSQLColumnBuilder: ColumnCreator {
+    func buildColumn(for column: Column, using queryBuilder: QueryBuilder) -> String? {
+        guard let type = column.type else {
+            return nil
+        }
+
+        var result = column.name
+        let identifierQuoteCharacter = queryBuilder.substitutions[QueryBuilder.QuerySubstitutionNames.identifierQuoteCharacter.rawValue]
+        if !result.hasPrefix(identifierQuoteCharacter) {
+            result = identifierQuoteCharacter + result + identifierQuoteCharacter + " "
+        }
+
+        var typeString = type.create(queryBuilder: queryBuilder)
+        if let length = column.length {
+            typeString += "(\(length))"
+        }
+        if column.autoIncrement {
+            guard let autoIncrementType = getAutoIncrementType(for: typeString) else {
+                //Unrecognised type for autoIncrement column, return nil
+                return nil
+            }
+            result += autoIncrementType
+        } else {
+            result += typeString
+        }
+
+        if column.isPrimaryKey {
+            result += " PRIMARY KEY"
+        }
+        if column.isNotNullable {
+            result += " NOT NULL"
+        }
+        if column.isUnique {
+            result += " UNIQUE"
+        }
+        if let defaultValue = column.defaultValue {
+            var packedType: String
+            do {
+                packedType = try packType(defaultValue, queryBuilder: queryBuilder)
+            } catch {
+                return nil
```

Comment:
Same as above.

Reply:
The nil is handled in the calling function and an error thrown that identifies the column as having invalid attributes.